pkg/providers/add-template/event.go

```diff
@@ -0,0 +1,51 @@
+package add_template
+
+import (
+	"github.com/google/go-github/v32/github"
+	"github.com/pingcap-incubator/cherry-bot/util"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"regexp"
+	"strings"
+)
+
+var (
+	templatePattern = regexp.MustCompile(`\/info`)
+)
+
+func (c *Comment) ProcessIssueCommentEvent(event *github.IssueCommentEvent) {
```

Comment:
When this function will be called

Reply:
`bot/webhook.go` may need to be updated.

Reply:
thanks, I will add this process in bot main function